flexprep/services/processing_service.py

```diff
@@ -0,0 +1,38 @@
+"""
+Services providing core functionality.
+"""
+
+import logging
+
+from flexprep import CONFIG
+from flexprep.domain.prepare_processing import PrepProcessing
+from flexprep.domain.s3_utils import S3client
+
+logger = logging.getLogger(__name__)
+
+
+def _check_s3input_content(s3_client, bucket_name: str) -> bool:
```

Comment:
Would move this function to s3_utils and perhaps rename as `_is_bucket_empty` since it only returns bool?